stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/data/user/User.kt

```diff
@@ -0,0 +1,13 @@
+package io.getstream.chat.ui.sample.data.user
+
+data class User(
```

Comment:
Could we use another name for this class?
It could be a bit confusing because we already have `User` in our SDK.
What about something like `DevelopmentUser` or any other name that reflects what it is?

Reply:
`SampleUser`?